audit_exceptions/github_prerelease_allowlist.json

```
@@ -57,5 +57,6 @@
   "virtualbuddy@beta": "all",
   "wezterm@nightly": "all",
   "xit": "all",
+  "yaak@beta": "all",
```

```suggestion
  "yaak@beta": "any",
```

This is required for updates to non-prerelease versions.